albumentations/core/transforms_interface.py

```diff
@@ -43,10 +43,21 @@ class CombinedMeta(SerializableMeta, ValidatedTransformMeta):
 
 
 class BasicTransform(Serializable, metaclass=CombinedMeta):
+    _targets: Union[Tuple[Targets, ...], Targets]  # targets that this transform can work on
+    _available_targets: Tuple[str, ...]  # targets that this transform, as string, lower-cased
+    _target2func: Dict[
+        str,
+        Callable[..., Any],
+    ]  # mapping for targets (plus additional targets) and methods for which they depend
```

Comment:
It's not same. `_available_targets` - same as _targets but str, lowercased. It will be used from compose to check targets, not calculate each time.
Name `_target2func` better change to `_key2func`  - here all targets plus additional names from additional_targets